src/ui/simulator/toolbox/components/datagrid/renderer/scenario-builder-hydro-levels-renderer.cpp

```diff
@@ -47,7 +47,7 @@ namespace Renderer
 	{
 		const double d = cellNumericValue(x, y);
 
-		return (d < 0.)
+		return (is_not_a_double(d))
```

Comment:
Check value with [`std::isnan`](https://en.cppreference.com/w/cpp/numeric/math/isnan).
This should work even if `d` was multipled by `100.0` in `cellNumericValue`